### DISCUSSION TITLE
Hardhat plugin for develop on Everscale

### DISCUSSION THREAD

**Discussion by ilyar** [everscale-org/bounties#20] [Ideas]
![image](https://user-images.githubusercontent.com/761285/187443509-51315202-af32-4d28-b2c7-41ea6f2a8f10.png)

Vote for the idea, this is a small step into the big world, well, I think so, they won’t rule out that there may be other opinions on this fog, feel free to say them in the comments, thanks.

https://hardhat.org/hardhat-runner/docs/advanced/building-plugins

> move from https://evernode.frill.co/b/8my22807/feature-ideas/hardhat-plugin-for-develop-on-everscale

**Comment by mnill** [comment]
At first there must be a research:
- Is it even possible?
- Will it be useful?
- Will it be better then locklift?
